README.md

```diff
@@ -135,14 +135,15 @@ socat TCP4-LISTEN:8081,fork,reuseaddr TCP4:A.B.C.D:8081
 
 #### Set up the SecureDrop Client
 1. Open a new terminal window.
-2. If it is not already installed, install Homebrew via the instructions at https://brew.sh/
-3. Install dependencies via Homebrew: `brew install pyenv gnupg oath-toolkit`
-4. Set up pyenv following the steps here: https://github.com/pyenv/pyenv#basic-github-checkout -  starting with #2 ("Configure your shell's environment for Pyenv"). You may need to close and reopen the terminal window for changes to be applied.
-5. install and select the latest version of python 3.9.x
-   ```
-   pyenv install 3.9.x
-   pyenv local 3.9.x
-   ```
+2. Install Homebrew
+  a. If you use an Intel based machine, follow the instructions at https://brew.sh/
+  b. If you use an Apple Silicon based machine (M1 or M2 Macs):
+    1. (Informational) If you already have a native `arm64` version of Homebrew, you will wind up with a separate Homebrew installation for each architecture. While they do not conflict, combining them in your `PATH` permanently may cause confusion about which brews are installed where. You may want to modify your `PATH` temporarily to perform this installation. The virtual environment you create in this session will continue to work in subsequent sessions.
+    2. Run `/usr/sbin/softwareupdate --install-rosetta` to allow you to run `x86_64` binaries
+    3. Enter a shell in `x86_64` mode with `arch -x86_64 bash`
+    4. Follow the instructions at https://brew.sh/
```

Comment:
I know all of these sub-items are part of "2. Install Homebrew" but this might be slightly clearer.
```suggestion
    4. Install Homebrew via the instructions at https://brew.sh/
```